tests/test_datamodule.py

```diff
@@ -55,8 +55,7 @@ def test_ogb_datamodule(self):
             rm(TEMP_CACHE_DATA_PATH, recursive=True)
 
         # Reset the datamodule
-        ds._data_is_prepared = False
-        ds._data_is_cached = False
+        ds = GraphOGBDataModule(task_specific_args, **dm_args)
```

Comment:
What's the purpose of this??

Reply:
This is properly resetting the `datamodule`. The old code didn't reset `self.train_ds`, etc., which could lead to wrong testing as, e.g., `setup` checks `if self.train_ds is None`

Reply:
But probalby we can remove this function and everything related to testing as @cwognum used `get_checkpoint_path` it in https://github.com/datamol-io/graphium/pull/441 where he has made some changes to testing :)